systemtests/utils/utils.go

```diff
@@ -154,6 +153,8 @@ func StartNetPlugin(t *testing.T, nodes []TestbedNode, nativeInteg bool) {
 
 // StartNetmasterWithFlags starts netplugin on specified testbed nodes with specified flags
 func StartNetmasterWithFlags(t *testing.T, node TestbedNode, flags map[string]string) {
+	time.Sleep(5 * time.Second)
```

Comment:
From what I noticed during my debugging, it appeared a general raft leader-election instability in the new environment. From consul logs I saw that the leader-election would happen time and again and if a test would run around one of those times had more chances of failing.

Since we don't change anything around etcd/consul configuration itself so starting/stopping netmaster shall really not have any timing affect. So I am not sure if the sleeps are really helping or we are just being lucky.

I don't know if we have a better option atm. May be we can track this separately then and try playing with etcd/consul configuration knobs around keepalives etc and see if that helps.